client/my-sites/domains/domain-management/add-google-apps/domains-select.jsx

```diff
@@ -7,6 +7,11 @@ import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 import React from 'react';
 
+/**
+ * Interal dependencies
```

Comment:
Typo 😉.

Reply:
That is what I get for not copying and pasting 😄